src/bin/pg_autoctl/monitor.c

```diff
@@ -635,21 +681,11 @@ monitor_register_node(Monitor *monitor, char *formation,
 	{
 		if (strcmp(parseContext.sqlstate, STR_ERRCODE_OBJECT_IN_USE) == 0)
```

Comment:
Shouldn't this use `monitor_retryable_error` too?
```suggestion
		if (monitor_retryable_error(parseContext.sqlstate, STR_ERRCODE_OBJECT_IN_USE))
```

Reply:
Yeah good point, we can check also for the generic monitor_retryable_error, and add the specific error that we can also retry during registration.